tests/logic/email_test.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import mock
+import unittest
+
+import logic.email
+
+
+class EmailTest(unittest.TestCase):
+    """We really just want to test that configuration is honored here."""
+
+    sender = 'test@example.com'
```

Comment:
Yep I see. Not requiring the sendgrid package when it‘s not needed makes perfect sense. 

What about moving `get_name_and_email` to util/email.py and add explicit tests for it?